addons/web_calendar/static/src/js/web_calendar.js

```diff
@@ -308,6 +308,12 @@ var CalendarView = View.extend({
                 self.proxy('update_record')(event._id, data);
             },
             eventRender: function (event, element, view) {
+                // List of event ids passed as 'highlight_event_ids' inside context will be high-lighted.
+                // This will provied developer the flexibility to high-light events based on custom
+                // requirements whenever displaying the calendar from any other view.
+                if (_.contains(self.dataset.context.highlight_event_ids, event.id)) {
+                    $(element).addClass("o_event_hightlight");
```

Comment:
No need to wrap element is jquery, it is already jquery element
http://fullcalendar.io/docs/event_rendering/eventRender/